book/from_flat_php_to_symfony2.rst

```diff
@@ -362,8 +373,12 @@ on the requested URI::
     }
 
 For organization, both controllers (formerly ``index.php`` and ``show.php``)
-are now PHP functions and each has been moved into a separate file, ``controllers.php``::
+are now PHP functions and each has been moved into a separate file named
+``controllers.php``. The job of each PHP function, now called a
+:term:`controller`, is to use information from the ``Request`` object to create
+and return a ``Response`` object.::
```

Comment:
either end with double colon (which will render as a single one) or end with a final stop (`.`) and use `.. code-block:: php` on a new line